### PR TITLE
Feature/7217 speedup bio keywords modal 3

### PR DESCRIPTION
**Pull request by amontenegro**

### CODE REVIEW

File: src/app/cdk/side-bar/modals/modal-keyword/modal-keyword.component.ts [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        visibility: new FormControl(this.defaultVisibility, {}),
+      })
+    )
+    this.keywords.push({
+      putCode: 'new-' + this.addedKeywordsCount,
+      visibility: { visibility: this.defaultVisibility },
+    } as Assertion)
+    this.addedKeywordsCount++
+
+    this._changeDetectorRef.detectChanges()
```
Comment (leomendoza123) [inline src/app/cdk/side-bar/modals/modal-keyword/modal-keyword.component.ts:162]: Autofocusing the new input is required

File: src/app/cdk/side-bar/modals/modal-keyword/modal-keyword.component.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  transition: transform 250ms cubic-bezier(0, 0, 0.2, 1);
+}
+
+.keyword-input-line {
+  margin-top: 20px;
+  img {
+    margin-top: 6px;
+    cursor: move;
+    z-index: 1;
+    margin-right: 18px;
```
Comment (leomendoza123) [inline src/app/cdk/side-bar/modals/modal-keyword/modal-keyword.component.scss:23]: Use `margin-inline-end: 18px;` to support arabian

Comment (leomendoza123) [reply src/app/cdk/side-bar/modals/modal-keyword/modal-keyword.component.scss:23]: I am doing this same fix for the other modals on my next PR

**Review by leomendoza123 — CHANGES_REQUESTED** [review]

File: src/app/cdk/side-bar/modals/modal-keyword/modal-keyword.component.ts [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+  }
+
+  formToBackend(keywordsForm: FormGroup): KeywordEndPoint {
+    const keywords = {
+      errors: [],
+      keywords: [],
+      visibility: this.originalBackendKeywords.visibility,
+    }
+    const toBackendKeywords = cloneDeep(this.keywords)
+    toBackendKeywords.reverse()
```
Comment (leomendoza123) [inline src/app/cdk/side-bar/modals/modal-keyword/modal-keyword.component.ts:109]: we need to call `_changeDetectorRef.detach()` before the reverse. 

This will avoid showing the list reversed on the UI while is been send to the backend. 

I have added this change to all the other modals on my last PR.

**Comment by amontenegro** [comment]
@leomendoza123 all fixed, please take a look again, thanks!

**Review by leomendoza123 — APPROVED** [review]

**Comment by amontenegro** [comment]
https://trello.com/c/g3CEJZAX/7217-speedup-bio-keywords-modal-3
